ui/Screen/Network.svelte

```diff
@@ -0,0 +1,180 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="typescript">
+  import { status } from "ui/src/localPeer";
+  import { indicatorState } from "ui/src/network";
+  import {
+    settings,
+    seedValidation,
+    addSeed,
+    removeSeed,
+  } from "../src/session";
+
+  import {
+    Button,
+    Icon,
+    SidebarLayout,
+    StyledCopyable,
+    TextInput,
+  } from "ui/DesignSystem";
+
+  let seedInputValue: string = "";
+
+  const submitSeed = async () => {
+    if (await addSeed(seedInputValue)) {
+      seedInputValue = "";
+    }
+  };
+
+  $: if (seedInputValue === "") {
+    seedValidation.reset();
+  }
+
+  let statusText: string = "";
+  let statusFill: string = "";
+  $: {
+    const state = indicatorState($status);
+    if (state) {
+      statusText = state.text;
+      statusFill = state.fill;
+    }
+  }
+</script>
+
+<style>
+  .container {
+    max-width: var(--content-max-width);
+    margin: 64px auto;
+    min-width: var(--content-min-width);
+    padding: 0 var(--content-padding);
+  }
+
+  section {
+    align-items: center;
+    margin-bottom: 24px;
+    padding: 0 12px;
+  }
+
+  .seed-entry-form {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .seed-entry-field {
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .seeds {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
```

Comment:
To accomplish a vertical layout I’d suggest
```suggestion
    display: flex;
    flex-direction: column;
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2066/commits/841cb0998cff274d0adfc34bb2b6b6bb458a8df9